osu.Framework/Game.cs

```diff
@@ -128,6 +128,19 @@ protected Game()
 
         protected sealed override void AddInternal(Drawable drawable) => throw new InvalidOperationException($"Use {nameof(Add)} or {nameof(Content)} instead.");
 
+        /// <summary>
+        /// The earliest point of entry during <see cref="GameHost.Run"/> starting execution of a game.
+        /// This should be used to set up any low level tasks such as exception handling.
+        /// </summary>
+        /// <remarks>
+        /// At this point in execution, only <see cref="Storage"/>
```

Comment:
accidentally the rest of the sentence I think

```suggestion
        /// At this point in execution, only <see cref="Storage"/> and <see cref="CacheStorage"/> are guaranteed to be valid for use.
```

Reply:
I swear my rider has a mind of its own at times.